packages/cosmic-swingset/src/chain-main.js

```diff
@@ -13,6 +13,8 @@ import { launch } from './launch-chain.js';
 import makeBlockManager from './block-manager.js';
 import { getTelemetryProviders } from './kernel-stats.js';
 
+let whenHellFreezesOver = null;
```

Comment:
lint gripes about unused variable; I presume you'll suppress it (and remove the setInterval global comment above)